tests/integration_tests/side_od_crop_augmentation.py

```diff
@@ -0,0 +1,71 @@
+import random
+from typing import Optional
+import numpy as np
+
+
+class DetectionRandomSideCrop:
+    """Crop a random part of the input.
+
+    Args:
+        height (int): height of the crop.
+        width (int): width of the crop.
+        p (float): probability of applying the transform. Default: 1.
+
+    Targets:
+        image, mask, bboxes, keypoints
+
+    Image types:
+        uint8, float32
+    """
+
+    def __init__(self, rel_x: float = 0.5, side: Optional[str] = None, p: float = 1.0):
+        if side is not None:
+            assert side in ["left", "right"], "side must be left or right"
+
+        assert rel_x >= 0 and rel_x <= 0.5, "rel_min_x must be between 0 and 0.5"
```

Comment:
aunque no es lo más común, en principio podrías quedarte con el 80% de la imagen.
```suggestion
        assert rel_x >= 0 and rel_x <= 1., "rel_min_x must be between 0 and 1"
```